robottelo/logging.py

```diff
@@ -19,50 +22,68 @@
 )
 
 
-logger = logzero.logger
-
-
-def configure_logzero():
-    """Modify the default logger instance"""
-    # TODO: remove console handler?
-    logfile = robottelo_log_dir.joinpath(f'{ROBOTTELO}.log')
-    logfile.parent.mkdir(parents=True, exist_ok=True)
-    logger.name = ROBOTTELO
-    logger.level = logging_yaml.robottelo.level
-    logger.logfile = logfile
-    logger.fileLoglevel = logging_yaml.robottelo.fileLevel
-    logger.formatter = defaultFormatter
-    logger.maxBytes = 10000000  # 10MB
-    logger.backupCount = 3
-
+logger = logzero.setup_logger(
+    name=ROBOTTELO,
+    level=logging_yaml.robottelo.level,
+    logfile=str(robottelo_log_file),
+    fileLoglevel=logging_yaml.robottelo.fileLevel,
+    isRootLogger=True,
+    formatter=defaultFormatter,
+    maxBytes=1e8,  # 100MB
+    backupCount=3,
+)
 
-configure_logzero()
 
-collection_logfile = robottelo_log_dir.joinpath(f'{ROBOTTELO}_collection.log')
-collection_logfile.parent.mkdir(parents=True, exist_ok=True)
+def configure_third_party_logging():
+    """Increase the level of third party packages logging."""
+    logger_names = (
+        'easyprocess',
+        'paramiko',
+        'requests.packages.urllib3.connectionpool',
+        'selenium.webdriver.remote.remote_connection',
+        'widgetastic_null',
+        'navmazing_null',
+        'airgun',
+        'nailgun',
+        'robozilla',
+    )
+
+    for logger_name in logger_names:
+        logger_levels = logging_yaml.get(logger_name) or logging_yaml.get('other')
+        logzero.setup_logger(
+            name=logger_name,
+            level=logger_levels['level'],
+            fileLoglevel=logger_levels['fileLevel'],
+            logfile=robottelo_log_file,
+            isRootLogger=False,
+        )
+
+
+configure_third_party_logging()
+
+
+# collection_logfile = robottelo_log_dir.joinpath(f'{ROBOTTELO}_collection.log')
+# collection_logfile.parent.mkdir(parents=True, exist_ok=True)
 collection_logger = logzero.setup_logger(
     name=f'{ROBOTTELO}.collection',
     level=logging_yaml.collection.level,
-    logfile=collection_logfile,
+    logfile=str(robottelo_log_file),
```

Comment:
do you need the explicit string conversion here?

Reply:
I don't think its _needed_, but its following the docblock for `logzero.setup_logger`, where this parameter is supposed to be a string file path.